### DISCUSSION TITLE
is there a shortcut to open autocomplete suggestions list?

### DISCUSSION THREAD

**Discussion by 5EGFAULT** [zed-industries/zed#8233] [Feature Requests]
is there a shortcut to open autocomplete suggestions list like **`⌥ + ESC`** in VS code?

**Comment by Moshyfawn** [comment]
It's `ctrl-space` in Zed by default.

lease note that MacOS may use this keymap for functions such as language switching. Therefore, you may want to either overwrite it with a different keymap or disable the system behaviour.

```json
[
  {
    "context": "Editor",
    "bindings": {
      "ctrl-space": "editor::ShowCompletions"
    }
  }
]
```

**Comment by xpe** [comment]
Be aware that macOS might steal `ctrl-space` via `System Settings` > `Input Sources` > `Select the previous input source`:

<img width="680" alt="image" src="https://github.com/zed-industries/zed/assets/4576329/7fe28bc1-bb07-4039-8402-7709bee54a6b">

**Reply by jamaluddinrumi** [reply]
thank you @xpe 
very helpful
i've stumbled upon these `ctrl space` problem about 3 months.
you just helped me a lot!

**Reply by alok8bb** [reply]
This setting is moved to `System Settings` > `Keyboard` > `Keyboard Shortcuts` > `Input Sources` in the newer Mac versions I think. Thanks! 

<img width="758" height="870" alt="image" src="https://github.com/user-attachments/assets/e677365d-fc5b-464d-85e1-4dba751f5125" />
